### PR TITLE
fix: [CDS-51663]: Handling Sync Status Response detail message

### PR DESCRIPTION
**Pull request by namantalaycha10**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
    - CodeFormat: `trigger codeformat`
    - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/43533)

### CODE REVIEW

File: 400-rest/src/main/java/software/wings/service/impl/instance/ContainerInstanceHandler.java [hunk 1]
```diff
@@ -924,6 +925,11 @@ private ContainerMetadata getContainerMetadataFromInstanceSyncResponse(DelegateR
       K8sTaskExecutionResponse k8sTaskExecutionResponse = (K8sTaskExecutionResponse) responseData;
       K8sInstanceSyncResponse syncResponse = (K8sInstanceSyncResponse) k8sTaskExecutionResponse.getK8sTaskResponse();
 
+      if (k8sTaskExecutionResponse.getCommandExecutionStatus().equals(FAILURE)) {
```
Comment (abosii-harness) [inline 400-rest/src/main/java/software/wings/service/impl/instance/ContainerInstanceHandler.java:928]: Can we move this after line 924

**Review by abosii-harness — CHANGES_REQUESTED** [review]

File: 400-rest/src/main/java/software/wings/service/impl/instance/ContainerInstanceHandler.java (hunk 1 above)
Comment (abosii-harness) [reply 400-rest/src/main/java/software/wings/service/impl/instance/ContainerInstanceHandler.java:928]: Can we handle this for helm sync reponse (before line 939)

**Review by namantalaycha10 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43533)* status: 2 file(s) to review. 1 change requests. 1. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`400-rest/src/main/java/software/wings/service/impl/instance/ContainerInstanceHandler.java` line 928 at r3](https://reviewable.io/reviews/harness/harness-core/43533#-NN_QNpGz337nj3EmmRG-r3-928:-NNf5odZ52nSZI7YznkY:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/4c6ace8c6dfcab310324c4d191dcc3e8b10632f4/400-rest/src/main/java/software/wings/service/impl/instance/ContainerInstanceHandler.java#L928)):*
<details><summary><i>Previously, abosii-harness (Alexandru Bosii) wrote…</i></summary><blockquote>

Can we handle this for helm sync reponse (before line 939)
</blockquote></details>

Done.

**Review by abhinavsingh1196 — APPROVED** [review]
> Reviewed 1 of 1 files at r4, 1 of 1 files at r5, 1 of 1 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43533)* status: 2 file(s) to review. 1 change requests. 1. unresolved discussions. 1 of 2 approvals obtained.

**Review by abosii-harness — APPROVED** [review]
> Reviewed 1 of 1 files at r4, 1 of 1 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/43533)* status: :shipit: complete! ✓

**Pull request merged**